lib/docs.js

```diff
@@ -36,17 +37,42 @@ module.exports = {
   init: function(controller, bot) {
     controller.hears('^docs (.*)', 'direct_message,direct_mention', (bot, message) => {
       let target = message.match[1];
+      let fuzzyResults = fuzzy.filter(target, availableDocs);
 
-      if (availableDocs.indexOf(target) > -1) {
-        bot.api.chat.postMessage({
-          channel: message.channel,
-          text: baseURL + target,
-          unfurl_links: false,
-          as_user: true
-        });
-      } else {
+      if (fuzzyResults.length == 0) {
```

Comment:
I didn't catch the fact that this file wasn't being linted. I'll add that and fix all the linter errors.